ducktape/__main__.py

```diff
@@ -0,0 +1,131 @@
+from ducktape.tests.test import TestLoader
+from ducktape.tests.test import SerialTestRunner
+from ducktape.tests.test import SimpleStdoutReporter
+from ducktape.tests.test import SimpleFileReporter
+from ducktape.tests.test import TestSessionContext
+from ducktape.cluster.vagrant import VagrantCluster
+
```

Comment:
There should probably be some documentation in here somewhere, at least a brief comment describing what code is in here.